src/renderer/components/BackgroundMaps/BackgroundMapInfo.js

```diff
@@ -0,0 +1,163 @@
+// @ts-check
+import { Button, Fade, makeStyles, Paper, Typography } from '@material-ui/core'
+import * as React from 'react'
+import { defineMessages, useIntl } from 'react-intl'
+import DeleteIcon from '@material-ui/icons/DeleteForeverOutlined'
+
+import Loading from '../Loading'
+import { useMapServerQuery } from '../../hooks/useMapServerQuery'
+import { MapboxPrevOnly } from './MapCard'
+
+const m = defineMessages({
+  // Title for Offline Areas
+  offlineAreas: 'Offline Areas',
+  // Button to create an offline area
+  createOfflineArea: 'Create Offline Area',
+  // Button to delete style
+  deleteStyle: 'Delete Style',
+  // Title for error message when deleting style
+  deleteErrorTitle: 'Error Deleting Style',
+  // Description for error message when deleting style,
+  deleteErrorDescription: 'There was an error deleting the style',
+  // Zoom Level Title
+  zoomLevel: 'Zoom Level: {zoom}',
+  // abbreviation for megabyte
+  mb: 'mb'
+})
+
+/**
+ * @typedef BackgroundMapInfoProps
+ * @prop {string} id
+ * @prop {string} idBeingViewed
+ * @prop {string} url
+ * @prop {number} size
+ * @prop {()=>void} unsetMapValue
+ */
+
+/** @param {BackgroundMapInfoProps} props */
+export const BackgroundMapInfo = ({
+  id,
+  idBeingViewed,
+  unsetMapValue,
+  url,
+  size
+}) => {
+  const shouldLoad = React.useMemo(() => id === idBeingViewed, [
+    id,
+    idBeingViewed
+  ])
+
+  const { formatMessage: t } = useIntl()
+
+  const { data } = useMapServerQuery(`/styles/${id}`, shouldLoad)
+
+  // Lazy loading each one here: aka will only load when clicked
+  return shouldLoad ? (
+    <Fade in={shouldLoad} timeout={600}>
+      <Paper
+        style={{
+          flex: 1,
+          width: '100%',
+          height: '100%',
+          padding: !data ? 40 : 0
+        }}
+      >
+        {!data ? (
+          <Loading />
+        ) : (
+          <React.Fragment>
+            <MapInfo
+              name={data.name}
+              id={id}
+              unsetMapValue={unsetMapValue}
+              url={url}
+            />
+            {/* Text */}
+            <div style={{ padding: 20 }}>
+              <Typography variant='subtitle2' style={{ fontSize: 18 }}>
+                {data.name}
+              </Typography>
+              {data.zoom && (
+                <Typography variant='body1'>
+                  {t(m.zoomLevel, { zoom: data.zoom })}
+                </Typography>
+              )}
+              <Typography variant='body1'>{`${size} ${t(m.mb)}`}</Typography>
```

Comment:
`size` needs to be converted to MB since it's in bytes up to this point

![image](https://user-images.githubusercontent.com/18542095/217063114-c2ac5b76-0278-4330-9421-420b1bd8ff1b.png)

Reply:
https://github.com/digidem/mapeo-desktop/pull/678/commits/35536ba7d93b16a61bd742dc226353e375e9499c